protocol/lib/ante/nested_msg.go

```diff
@@ -2,73 +2,99 @@ package ante
 
 import (
 	"fmt"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authz "github.com/cosmos/cosmos-sdk/x/authz"
 	gov "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/dydxprotocol/v4-chain/protocol/lib/metrics"
 )
 
+const DYDX_MSG_PREFIX = "/dydxprotocol"
```

Comment:
mega nit: might be better to use the [AppName](https://github.com/dydxprotocol/v4-chain/blob/4357938804d73e46bf86ab16c28bb983720c2bcd/protocol/app/constants.go#L4)